benches/proof.rs

```diff
@@ -32,7 +32,7 @@ pub fn bench_zk_proof_prove_and_verify_1024(b: &mut Bencher) {
 
     b.iter(|| {
         let proof_results = Paillier::prove(&dk, &challenge);
-        let _result = Paillier::verify(&proof_results.unwrap(), &verification_aid);
+        let _result = Paillier::verify_correctKey(&proof_results.unwrap(), &verification_aid);
```

Comment:
with https://github.com/mortendahl/rust-paillier/pull/25 you shouldn't need to do this anymore! you'll simply have to explicitly choose the proof needed, ie `use paillier::proof::CorrectKey;`, and it'll know which `verify` you're thinking about

Reply:
Thanks, I will merge #25. I was wandering how come it compiles on @gbenattar machine without renaming 'verify'

Reply:
👍